Source/Invoke-PackerBuild.ps1

```diff
@@ -184,26 +190,28 @@ switch ($PSCmdlet.ParameterSetName) {
                 "ish_mock_connectionstring=$MockConnectionString"
 	        )
         }
+        $packerFileLocation="Vagrant\$ServerVersion"
 
-        $packerFileNameName="ish-$ServerVersion-vagrant-hyperv-iso.json"
+        $packerFileName="ish-$ServerVersion-vagrant-hyperv-iso.json"
         $logRegExSource="hyperv-iso"
     }
 }
 
-Write-Host "Using $packerFileNameName"
+Write-Host "Using $packerFileName"
 
-$packerArgs+=$packerFileNameName
+$packerArgs+=$packerFileName
 
-Push-Location -Path "$PSScriptRoot\Packer" -StackName Packer
+$pushPath=Join-Path -Path "$PSScriptRoot\Packer" -ChildPath $packerFileLocation
+Push-Location -Path $pushPath -StackName Packer
 
 try
 {
     $invokedPacker=$false
 
-    if ($PSCmdlet.ShouldProcess($packerFileNameName, "packer build")){
+    if ($PSCmdlet.ShouldProcess($packerFileName, "packer build")){
         $invokedPacker=$true
         $env:PACKER_LOG=1
-        $packetLogPath=Join-Path $env:TEMP "$($packerFileNameName).txt"
+        $packetLogPath=Join-Path $env:TEMP "$($packerFileName).txt"
```

Comment:
Unfortunately ISHBootstrap's builders depends only on public artifacts and that means only ISH.12.0.3 at this moment